.gitignore

```diff
@@ -0,0 +1 @@
+.md
```

Comment:
- [ ] Please, consider adding exactly the `test.md` file into the `.gitignore` file, as required in the project, to ignore it while you will pushing.